ubuntudesign/cloud-init
# Second pass at cloud init pages

## Done

- added correct copy
- added correct partner logos
- added correct images of contributors
- tweaked layout

## QA

Again, this is not ready for a visual QA, just a code sanity check. Please ignore where there are `//TODO` comments are what's below will be refactored before release. 

This version will be pushed onto a `gh-pages` branch so I can step through the build with the design team to finesse.
